kotlinx-coroutines-core/common/src/internal/StackTraceRecovery.common.kt

```diff
@@ -40,6 +40,7 @@ internal expect suspend inline fun recoverAndThrow(exception: Throwable): Nothin
  * The opposite of [recoverStackTrace].
  * It is guaranteed that `unwrap(recoverStackTrace(e)) === e`
  */
+@PublishedApi
```

Comment:
Please leave a comment that it is only used from other module tests, so when the time comes we know it can be safely deleted/reverted